views/services/update.go

```diff
@@ -180,100 +179,120 @@ func (m *Model) SetContent(msg Msg) {
 }
 
 func (m *Model) setRenderItem() {
-	const replicaWidth = 10
-	const statusWidth = 12
-	const createdWidth = 10
-	const updatedWidth = 10
-	const gap = 2
-
+	// We'll partition the available width into equal columns (6 columns):
+	// SERVICE | STACK | REPLICAS | STATUS | CREATED | UPDATED
 	width := m.List.Viewport.Width
 	if width <= 0 {
 		width = 80
 	}
 
-	// Compute the longest service and stack names in the filtered list
-	maxService := len("SERVICE")
-	maxStack := len("STACK")
-	for _, e := range m.List.Filtered {
-		if len(e.ServiceName) > maxService {
-			maxService = len(e.ServiceName)
+	cols := 6
+	starts := make([]int, cols)
+	for i := 0; i < cols; i++ {
+		starts[i] = (i * width) / cols
+	}
+	colWidths := make([]int, cols)
+	for i := 0; i < cols; i++ {
+		if i == cols-1 {
+			colWidths[i] = width - starts[i]
+		} else {
+			colWidths[i] = starts[i+1] - starts[i]
 		}
-		if len(e.StackName) > maxStack {
-			maxStack = len(e.StackName)
+		if colWidths[i] < 1 {
+			colWidths[i] = 1
 		}
 	}
```

Comment:
Third time that I see this kind of code now. The header should be integrated into the FilteredList itself: https://github.com/mosonyi/swarmcli/issues/150